### PR TITLE
Add new plugin: Locations and Areas

### PR DESCRIPTION
**Pull request by 100wunder**
I added my WordPress Plugin "Locations and Areas" to reach a bigger audience. It extends basic map functionality by adding Areas as additional map navigation. It is also possible to allow website visitors adding their own locations directly from the frontend. The WordPress community needs more Leaflet.js!

### CODE REVIEW

**Review by jonkoops — APPROVED** [review]
Looks good but there are some conflicts that will need to be resolved.

**Comment by 100wunder** [comment]
@jonkoops I fixed the merge conflict. Thanks for your support!

**Review by Falke-Design — APPROVED** [review]

File: docs/plugins.md
Comment (Falke-Design) [inline docs/plugins.md:4996]: Please replace all blanks with tabs
```suggestion
	<td>
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]

File: docs/plugins.md
Comment (johnd0e) [inline docs/plugins.md:4995]: Tabs

File: docs/plugins.md
Comment (johnd0e) [inline docs/plugins.md:5013]: Tabs

**Review by johnd0e — CHANGES_REQUESTED** [review]

File: docs/plugins.md
Comment (Falke-Design) [inline docs/plugins.md:5001]: Please move `<td>` to a new line

File: docs/plugins.md
Comment (Falke-Design) [inline docs/plugins.md:4999]: Please move `<td>` to a new line

File: docs/plugins.md
Comment (Falke-Design) [inline docs/plugins.md:4987]: Use tabs

File: docs/plugins.md
Comment (Falke-Design) [inline docs/plugins.md:4986]: Use tabs

File: docs/plugins.md
Comment (Falke-Design) [inline docs/plugins.md:4976]: Use tabs

File: docs/plugins.md
Comment (Falke-Design) [inline docs/plugins.md:4973]: Use tabs

**Review by Falke-Design — CHANGES_REQUESTED** [review]

**Review by 100wunder — COMMENTED** [review]
@Falke-Design Thanks for your suggestions! I added consistency with tabs and table structure before already and currently wait for @johnd0e approval. It's always possible to optimize further but IMO this should be topic of a future PR.

**Comment by Falke-Design** [comment]
@100wunder you need to change your PR. You can replace the blanks with tabs or you remove the changes that are not necessary for you.

**Comment by 100wunder** [comment]
@Falke-Design okay I did

**Comment by 100wunder** [comment]
@johnd0e If you find some time would you please review? I had send you a review request.

**Comment by Falke-Design** [comment]
We added the plugin in the refactoring where we moved the plugins into seperate files #7805

**Comment by 100wunder** [comment]
Thanks @Falke-Design, @jonkoops and @johnd0e for your contributions and support.

**Pull request closed without merging**
